app/assets/v2/js/pages/leaderboard.js

```diff
@@ -1,19 +1,20 @@
 /* eslint-disable no-invalid-this */
 
 $( document ).ready( function() {
-  $( '.leaderboard_entry .progress-bar' ).each( function (){
+  $( '.leaderboard_entry .progress-bar' ).each( function() {
     const max = parseInt( $( this ).attr( 'aria-valuemax' ) );
     const now = parseInt( $( this ).attr( 'aria-valuenow' ) );
     const width = ( now * 100 ) / max;
+
     $( this ).css( 'width', `${width}%` )
   })
 
-  $( '.clickable-row' ).click( function(e){
+  $( '.clickable-row' ).click( function( e ) {
```

Comment:
@mbeacom are we fixed on this format ? 
 It looks odd `( foo )` as compared to `(foo)`